### PR TITLE
Generated code for clusters is using the wrong API group

### PR DESCRIPTION
**Pull request by viswanathtadi**
# Description

Fixes VZ-2865

# Checklist 

As the author of this PR, I have:

- Checked that I included or updated copyright and license notices in all files that I altered

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

**Review by markxnelson — APPROVED** [review]

File: application-operator/apis/clusters/v1alpha1/doc.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// Copyright (c) 2020, 2021, Oracle and/or its affiliates.
```
Comment (mgianatagh) [inline application-operator/apis/clusters/v1alpha1/doc.go:1]: If this is a new file, the copyright should just be "2021,"

File: application-operator/apis/oam/v1alpha1/doc.go [hunk 2]
```diff
@@ -0,0 +1,7 @@
+// Copyright (c) 2020, 2021, Oracle and/or its affiliates.
```
Comment (mgianatagh) [inline application-operator/apis/oam/v1alpha1/doc.go:1]: Same comment as above, I think the copyright year should just be "2021,"

File: platform-operator/apis/clusters/v1alpha1/doc.go [hunk 3]
```diff
@@ -0,0 +1,7 @@
+// Copyright (c) 2020, 2021, Oracle and/or its affiliates.
```
Comment (mgianatagh) [inline platform-operator/apis/clusters/v1alpha1/doc.go:1]: copyright year is "2021,"

File: platform-operator/apis/verrazzano/v1alpha1/doc.go [hunk 4]
```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2020, 2021, Oracle and/or its affiliates.
```
Comment (mgianatagh) [inline platform-operator/apis/verrazzano/v1alpha1/doc.go:1]: copyright year is "2021,"

**Review by mgianatagh — APPROVED** [review]

**Comment by viswanathtadi** [comment]
@mgianatagh can I merge it now?

**Pull request closed without merging**